chalicelib/checks/wfr_cgap_checks.py

```diff
@@ -10,7 +10,8 @@
 
 # list of acceptible version
 cgap_partI_version = ['WGS_partI_V11', 'WGS_partI_V12', 'WGS_partI_V13']
```

Comment:
Part I should also use either v14 or v15 (we fixed add readgroup)

Reply:
(but we don't need to rerun if say v13 has already run and didn't cause a following step to fail)